public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -218,7 +218,7 @@ export class PanelQueryRunner {
   }
 
   /**
-   * Called after every streaming event.  This should be throttled so we
+   * Called after every streaming event. This should be throttled so we
```

Comment:
ha!  https://www.cultofpedagogy.com/two-spaces-after-period/

I am in fact over 40 :)

Reply:
I heard about that but did not realise that could be the case, thought it was a simple typo :D